authority/voting/server/wire_handler.go

```diff
@@ -37,8 +38,17 @@ func (s *Server) onConn(conn net.Conn) {
 	)
 
 	rAddr := conn.RemoteAddr()
-	s.log.Debugf("Accepted new connection: %v", rAddr)
+	var proto string
+	switch conn.(type) {
+	case *net.TCPConn:
+		proto = "tcp"
```

Comment:
does there need to be a tcp6 case here? i guess not. there's only a net.TCPConn type.

Reply:
correct